### ISSUE TITLE
Matrix and evaluation of a better model in classification

### ISSUE THREAD

**Issue by shangguan9191** [nicolas-chaulet/torch-points3d#425]
I am doing the comparison of different models.  Normally, I see OA comparison in modelnet40. But how is OA evaluated is not specified. Is it average over epochs or best accuracy at certrain epoch?  And Why dont they care how much epoch they have used and total training time spent.e.g time performance?

**Comment by nicolas-chaulet** [comment]
Those are good questions and it probably depends on each author! Our checkpoints contain a few models every time you run a training, the one best for loss, overall accuracy and latest so that you can choose which one to use for inference.

**Comment by shangguan9191** [comment]
Can you specify the Loss? Is it comparison between train loss and Val
loss. Or it is the loss alone. Why there are always best accuracy result
compared in paper instead of loss. And how about the matrix training time
per epoch for time performance


Nicolas Chaulet <notifications@github.com> 于 2020年9月3日周四 下午7:10写道：

**Comment by nicolas-chaulet** [comment]
By default selection in TP3d is on the validation set. 

> Why there are always best accuracy result
compared in paper instead of loss

Could you pelase clarify?

Not sure what you mean by matrix training time. The timing of each epoch gets printed in the logs automatically.
